### PR TITLE
Victor stone

### PR DESCRIPTION
**Pull request by victor-stone**
Please use this version of the article. (Still needs an editor's eye)

### CODE REVIEW

File: posts/2017-09-07-big-changes-for-a-small-org.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
+---
+title: "Preparing for Big Scale in a Small Shop Using Serverless"
+description: When a group of scrappy activists raise millions of dollars, they need an architecture that never turns a donor away
+date: 2017-09-07
+layout: Post
+thumbnail: https://movementvote.org/images/Movement-2018-logo-blue.png
```
Comment (andrea-p) [inline posts/2017-09-07-big-changes-for-a-small-org.md]: @victor-stone If you have a square image to go here, it'll make sure the thumbnail doesn't show up squished

**Comment by andrea-p** [comment]
@victor-stone Edited! Let me know what you think, and/or if I accidentally made any of the information incorrect with my edits. If all is good, we'll plan to publish this on Thursday morn!

**Comment by victor-stone** [comment]
Picture url is updated - let me know if that doesn't work

**Comment by andrea-p** [comment]
schedule(09/07/2017 6:00am)

**Comment by DavidWells** [comment]
✅ Post scheduled for **09/07/2017 at 6:00amundefined (America/Los_Angeles)**

**To unscheduled the post:**

1. **[Delete the previous comment ⬆](https://github.com/serverless/blog/pull/288#issuecomment-327640248)** with schedule(time) information by clicking the ❌
  ```
  ┌──────────────────────────────────────────────┐
  │                                           X  │
  ├───────────────────────────────────────────▲──┤
  │                                           │  │
  │  schedule(mm/dd/yyyy h:mm A)   ┌────────┐ │  │
  │                                │ Delete ├─┘  │
  │                                └────────┘    │
  └──────────────────────────────────────────────┘
  ```
2. (Optionally) Delete this comment to keep things tidy

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request merged**